src/SonarScanner.MSBuild.Common/CommandLine/CommandLineFlagPrefix.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SonarScanner.MSBuild.Common.CommandLine
+{
+    public static class CommandLineFlagPrefix
+    {
+        private static readonly char[] _prefixes = new[] { '-', '/' };
+
+        public static string[] GetPrefixFlag(string[] flags)
+        {
+            var flagPrefixed = new List<string>();
+            foreach (var flag in flags)
+            {
+                foreach (var prefix in _prefixes)
+                    flagPrefixed.Add($"{prefix}{flag}");
```

Comment:
The convention in all of our C# is to always use `{ ... }`, even for single lines.

Reply:
Ok, that has been fixed.